services/getstream/GetstreamService.js

```diff
@@ -12,7 +11,7 @@ class GetstreamService {
     const query = {
       limit,
       reactions: {own: true, recent: true, counts: true},
-      // ranking: GETSTREAM_RANKING_METHOD,
+      ranking: GETSTREAM_RANKING_METHOD,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Uncommenting the `ranking` property and setting its value to `GETSTREAM_RANKING_METHOD` is a good change if you want to apply a specific ranking method when fetching feeds. Just make sure that `GETSTREAM_RANKING_METHOD` is properly defined and holds the correct value for your use case.
```diff
- // ranking: GETSTREAM_RANKING_METHOD,
+ ranking: GETSTREAM_RANKING_METHOD,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->